docs/eventModelling.md

```diff
@@ -0,0 +1,48 @@
+Firewall prevents a device talking to network by dropping ICMP Ping request.|```<Network>``` with sub-element ```<Deny>```.|It is presumed that this is a routine and fully automatic activity.  If this was not the case then ```<Alert>``` might be more appropriate.
+Host based, software firewall allows a process to listen on a port.|```<Network>``` with sub-element ```<Listen>```
+User badges into a building.|```<Authenticate>```|*There will be a new schema action for physical access / physical presence events in a future version of the schema*
+User attempts to badge into a building but is denied as they are not a member of the necessary group.|```<Authenticate>```|*There will be a new schema action for physical access / physical presence events in a future version of the schema*
+User saves a draft email.|```<Create>``` with an ```<Object>``` of type ```<Email>```
+User clicks on "Export results to XLS" within an application.|```<Export>```with an ```<Object>``` of type ```<File>```|This is an ```<Export>``` as it relates to information being transferred out of a controlled area (the application).
+User clicks on "Print to PDF" within an application.|```<Export>```with an ```<Object>``` of type ```<File>```|N.B. not ```<Print>```, as it doesn't relate to creating hard-copy, but it is an ```<Export>``` as it relates to information being transferred out of a controlled area (the application).
+Web proxy allows an authenticated user access to a website of known category.|```<View>``` of type ```<Resource>``` that has ```<Category>``` element assigned to that reported by the web proxy|Events are modelled from the point of view of the user, where possible.
+Someone plugs a USB drive into a workstation while a user session is logged in.|```<Install>``` of type ```<Media>```.  ```<User>``` is the user currently logged into the workstation.|
+Someone unplugs a USB drive into a workstation while a user session is not logged in.|```<Uninstall>``` of type ```<Media>```.  There is no  ```<User>```  element.|
+Antivirus product on a workstation with a logged in user detects malware on a file.|```<AntiMalware>``` with sub-element ```<Quarantine>``` |*This will be modelled as ```<Alert>``` in a future version of schema.*
+An IPS appliance blocks an IP address that may be attempting an SQL injection attack.|```<Alert>``` with sub-element ```<IDS>```.  Attacker IP is recorded in ```<Source>``` and target in ```<Destination>```.|
+A database reports that a processing account logs on and performs a backup.|```<Authenticate>``` and then a second event, ideally ```<Process>``` if the backup is contained within the database and control maintained, otherwise ```<Export>```.| It might be unclear that this is a backup operation, e.g. perhaps a query is performed.  In this case ```<Search>``` may be most appropriate for the second event.
```

Comment:
could do export where src is Criteria and dest is the backup file

Reply:
in #26 changed to:
In this case `<Criteria>` could be used as the `<Source>` of the `<Export>`